ENV_VARIABLES.md

```diff
@@ -0,0 +1,41 @@
+# [2wp-app] Environment Variables
+This table was created to guide and centralize the **environment variables** necessary for correct execution.
+The value of these variables are used in **environment-variables.ts** file.
+
+|NAME                                        |DEV VALUE                      |DETAILS                                                  |
```

Comment:
Some "DEV VALUE"s are confused:
Empty values or only \

Reply:
gotta agree with @leoiovlabs 
There are some ` `` ` that don't make much sense, is it trying to represent `""` ?

Reply:
removed '' values, and unused variables like BASE_URL=/